packages/mobile/src/app/App.tsx

```diff
@@ -66,6 +67,23 @@ export class App extends React.Component<Props> {
   async componentDidMount() {
     await ValoraAnalytics.init()
 
+    // Handles opening Clevertap deeplinks when app is closed / in background
+    CleverTap.getInitialUrl(async (err: any, url: any) => {
+      if (err) {
+        Logger.error('App CleverTap Deeplink on Load', err)
```

Comment:
The error argument is the 3rd one.
```suggestion
        Logger.error('App/componentDidMount', 'App CleverTap Deeplink on Load', err)
```

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2012/commits/f5ea9f621cfeb3bf2541fccf39c03e29335f24e1!